libs/go/zmscli/template.go

```diff
@@ -46,6 +46,7 @@ func (cli Zms) ShowServerTemplate(templateName string) (*string, error) {
 	}
 	var buf bytes.Buffer
 	buf.WriteString("template:\n")
+	cli.dumpMetadata(&buf, template.Metadata ,indent_level2_dash, templateName)
```

Comment:
fix the comma before and after the space for template.Metadata

gofmt -d template.go

will show all the formatting errors. not sure why this is not being flagged during the build since we do call gofmt.

Reply:
Thank you. Done.